lms/djangoapps/discussion/rest_api/tests/test_tasks.py

```diff
@@ -230,3 +239,224 @@ def test_notification_is_send_to_cohort_ids(self, cohort_text, notification_type
         self.assertEqual(handler.call_count, 1)
         user_ids_list = [user.id for user in audience]
         self.assert_users_id_list(user_ids_list, handler.call_args[1]['notification_data'].user_ids)
+
+
+@override_waffle_flag(ENABLE_NOTIFICATIONS, active=True)
+class TestSendResponseNotifications(DiscussionAPIViewTestMixin, ModuleStoreTestCase):
+    """
+    Test for the send_response_notifications function
+    """
+
+    def setUp(self):
+        super().setUp()
+        httpretty.reset()
+        httpretty.enable()
+
+        self.course = CourseFactory.create()
+        self.user_1 = UserFactory.create()
+        CourseEnrollment.enroll(self.user_1, self.course.id)
+        self.user_2 = UserFactory.create()
+        CourseEnrollment.enroll(self.user_2, self.course.id)
+        self.user_3 = UserFactory.create()
+        CourseEnrollment.enroll(self.user_3, self.course.id)
+        self.thread = ThreadMock(thread_id=1, creator=self.user_1, title='test thread')
+        self.thread_2 = ThreadMock(thread_id=2, creator=self.user_2, title='test thread 2')
+        self.thread_3 = ThreadMock(thread_id=2, creator=self.user_1, title='test thread 3')
+        for thread in [self.thread, self.thread_2, self.thread_3]:
+            self.register_get_thread_response({
+                'id': thread.id,
+                'course_id': str(self.course.id),
+                'topic_id': 'abc',
+                "user_id": thread.user_id,
+                "username": thread.username,
+                "thread_type": 'discussion',
+                "title": thread.title,
+            })
+
+    def test_basic(self):
+        """
+        Left empty intentionally. This test case is inherited from DiscussionAPIViewTestMixin
+        """
+
+    def test_not_authenticated(self):
```

Comment:
can we not simply avoid writing these, instead of leaving them empty?

Reply:
It will fail in this particular case. These are being inherited from the mixin and are not needed in our case. Added empty cases to avoid test case failure.